### PR TITLE
View

### PR DESCRIPTION
**Pull request by lianthompson**
UPDATES:

- Remove Navbar except for Logo
- Add background illustration and Phase Zero & Unstack
- Add shadow to Phase Zero and Unstack
- Change Zero Phase to Phase Zero

TO DO:

- Add background waves
- Update images
- Add Navbar
- Add companies we've worked with
screen shot 2018-08-17 at 10 01 03 am
screen shot 2018-08-17 at 10 01 10 am

### CODE REVIEW

File: gatsby-site/src/components/Projects/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import './projects.css';
+
+const Projects = (props) => {
+  return (
+    <div className="project-container">
+        <div className="project">
+        <img className="image" src='https://res.cloudinary.com/colark/image/upload/v1534454636/Colark%20Marketing%20Site/Colark_Marketing_Site_PhaseZeroEdited.png'/>
```
Comment (alodahl) [inline gatsby-site/src/components/Projects/index.js]: Minor indenting fixes here 
<- div
<-/div
<- div
<- /div

File: gatsby-site/.gitignore [hunk 2]
```diff
@@ -2,6 +2,7 @@
 .cache
 node_modules
 yarn-error.log
+package-lock.json
```
Comment (alodahl) [inline gatsby-site/.gitignore]: Ellie told me we should keep the package-lock after all!  I'm going to remove this from my gitignore.

Comment (lianthompson) [reply gatsby-site/.gitignore]: Awesome - I took it out of mine too.

File: gatsby-site/src/components/Navbar/navbar.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    color: var(--colarkfontcolor);
+    font-size: 1.5vw;
+    font-family: var(--colarkfont);
+}
+
+span.navbar-toggler-icon {
+    font-size: 24px;
+}
+
+button {
```
Comment (alodahl) [inline gatsby-site/src/components/Navbar/navbar.css]: I would name this button with a more specific class if I were you... unless literally every "button" on the site is going to look like this. Even if they were, I'd still call it "main button" or something.

Very clean PR, nice work!

File: gatsby-site/src/components/Navbar/navbar.css [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    --colarkfont: 'Mukta Malar', sans-serif;
+    --background: #FFFFFF;
+}
+
+.sticky-nav {
+    top: 0;
+    padding-left: 100px;
+    width: 100% !important;
+    border-bottom: 1px solid #C4C4C4;
+
```
Comment (alodahl) [inline gatsby-site/src/components/Navbar/navbar.css]: Remove this empty line while you're at it =)

File: gatsby-site/src/components/Navbar/navbar.css [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+button {
+    background-color: #FF6969;
+    color: #FFFFFF;
+    border-radius: 5px;
+    font-size: 24px !important;
+    height: var(--navheight);
+    padding-bottom: 5px !important;
+
```
Comment (alodahl) [inline gatsby-site/src/components/Navbar/navbar.css]: This one too

File: gatsby-site/src/layouts/index.js [hunk 6]
```diff
@@ -40,12 +45,11 @@ const Layout = ({ children, data }) => (
         { name: 'pinterest-rich-pin', content: 'false' }
       ]}
     />
-    <ReactNavbar siteTitle={data.site.siteMetadata.title} />
+    {/* <ReactNavbar siteTitle={data.site.siteMetadata.title} /> */}
```
Comment (alodahl) [inline gatsby-site/src/layouts/index.js]: Do we need to keep this? I assume this is the same component that is now in View/index.js

File: gatsby-site/src/layouts/index.js [hunk 7]
```diff
@@ -6,13 +6,18 @@ import 'bootstrap/dist/css/bootstrap.min.css';
...
 // import './bootstrap.min.css';
 
-import ReactNavbar from '../components/header'
+// import ReactNavbar from '../components/Navbar';
+import View from '../components/View';
 import './index.css'
 
 const Layout = ({ children, data }) => (
-  <div>
+  <div className="testing">
```
Comment (alodahl) [inline gatsby-site/src/layouts/index.js]: what does "testing" mean?

**Pull request merged**